diff --git a/core/config.rb b/core/config.rb
index 15ac9308..20234be9 100644
--- a/mikutter/core/config.rb
+++ b/mikutter/core/config.rb
@@ -14,8 +14,8 @@ module CHIConfig
   ACRO = "mikutter"
 
   # 下の２行は馬鹿にしか見えない
-  TWITTER_CONSUMER_KEY = "AmDS1hCCXWstbss5624kVw"
-  TWITTER_CONSUMER_SECRET = "KOPOooopg9Scu7gJUBHBWjwkXz9xgPJxnhnhO55VQ"
+  TWITTER_CONSUMER_KEY = "rcktNVY9J20xh6JXFajFa519s"
+  TWITTER_CONSUMER_SECRET = "SJOLJ7ePNOVN4zEIpyXIJXTARGBzw34yKFXXjogk1Cq2kuugnb"
   TWITTER_AUTHENTICATE_REVISION = 1
 
   # pidファイル
